src/test/java/org/ow2/proactive/catalog/service/CatalogObjectServiceTest.java

```diff
@@ -72,6 +72,8 @@ public class CatalogObjectServiceTest {
 
     public static final String COMMIT_MESSAGE = "commit message";
 
+    public static final String UPDATE_COMMIT_MESSAGE = "The project name or/and tags metadata are updated";
```

Comment:
why define this constant in the Test class ? you are using the message in the main code also

Reply:
you should import the constant instead of redefining it